www/index.html

```diff
@@ -359,6 +359,7 @@
   <script type="text/javascript" src="lib/framework7/js/framework7.min.js"></script>
   <script type="text/javascript" src="js/init-styles.js"></script>
   <script type="text/javascript" src="js/my-app.js"></script>
+  <script type="text/javascript">init();</script>
```

Comment:
Yeah, the typical way just seems to be, go for it, and the framework will sort it out.
Given your specific requirements you're probably fine.
Is it okay that `var mainView` has been removed from the global scope and now only exists inside the init function?